server/api/users.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express')
+const router = express.Router()
+
+const {
+    models: { User, Review },
+} = require('../db')
+
+router.get('/', async (req, res, next) => {
+    const users = await User.findAll({ include: Review })
+    res.send(users)
+})
+//get single user by specifying their email address
+router.get('/:email', async (req, res, next) => {
+    const user = await User.findOne({
+        where: {
+            email: req.params.email,
+        },
+    })
+    res.send(user)
+})
+
```

Comment:
Sorry, just a quick question: Why not just using 'router.get('/:id)'??

Reply:
I did that because the id is a uuid now so i did that just to simplify it, but now that you mention it... think I'd better switch it back to 'users/:id' lol. Sorry, not very secure.